api/src/test/java/io/opentelemetry/trace/util/SamplersTest.java

```diff
@@ -32,7 +32,7 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** Unit tests for {@link Samplers}. */
+/** Unit tests for {@link io.opentelemetry.trace.util.Samplers}. */
```

Comment:
why does this change to fully qualify?

Reply:
There is no reason. My IDE changed this to fully qualified name when moving the class.